docs/tutorial/kubernetes-charm-django.rst

```diff
@@ -785,6 +778,17 @@ And then you can proceed with its deletion:
     multipass delete charm-dev
     multipass purge
 
+If you'd like to manually reset your working environment, you can run the
+following in the rock directory ``~/django-hello-world`` for the tutorial:
```

Comment:
Note for work in the next cycle: In other tutorials I made this a required step, even though it was optional. We're not to fork in tutorials.